sw/device/tests/otbn_randomness_test.c

```diff
@@ -27,6 +31,121 @@ static const otbn_addr_t kVarUrndOut = OTBN_ADDR_T_INIT(randomness, urnd_out);
 
 const test_config_t kTestConfig;
 
+static dif_clkmgr_t clkmgr;
+const dif_clkmgr_hintable_clock_t otbn_clock =
+    kTopEarlgreyHintableClocksMainOtbn;
+
+/**
```

Comment:
I don't think you actually need these comments. They're not public API and quite self-evident (and the code above doesn't actually bother)